src/components/AvatarWithIndicator.js

```diff
@@ -0,0 +1,47 @@
+import React, {memo} from 'react';
+import {View, StyleSheet} from 'react-native';
+import PropTypes from 'prop-types';
+import Avatar from './Avatar';
+import styles from '../styles/styles';
+
+const propTypes = {
+    // Is user active?
+    isActive: PropTypes.bool,
+
+    // url for the avatar
+    source: PropTypes.string.isRequired,
+
+    // Extra styles to pass
+    style: PropTypes.arrayOf(PropTypes.any),
+};
+
+const defaultProps = {
+    isActive: false,
+    style: [],
+};
+
+const AvatarWithIndicator = ({
+    isActive,
+    source,
+    style,
+}) => {
+    const indicatorStyles = [
+        styles.statusIndicator,
+        styles.statusIndicatorBottomRight,
```

Comment:
Since there is only one position the bottom right can be added to the `statusIndicator` style.

Reply:
We can now remove this because it is `undefined`.